ArduCopter/mode_guided.cpp

```diff
@@ -802,4 +802,9 @@ float ModeGuided::crosstrack_error() const
     }
 }
 
+bool ModeGuided::use_pilot_yaw(void) const
```

Comment:
maybe put a comment above this method.  Also let's make sure the order in the .cpp matches the .h order

Reply:
Pretty much all methods get a comment even if it is very simple.  So let's add a comment, "return true if pilot's yaw input should be used to adjust vehicle's heading"

Reply:
Done.